<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>IDT_C2</class>
 <widget class="QWidget" name="IDT_C2">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>487</width>
    <height>164</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>IDT_C2</string>
  </property>
  <widget class="QTextBrowser" name="TBx_Messages">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>231</width>
     <height>81</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="CB_Receivers">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>40</y>
     <width>69</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="B_SendScreenShot">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>130</y>
     <width>101</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Select And Send</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>40</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>Port Select</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>151</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>System Messages</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
